sdk/build-scripts/src/build-dapp-kit.ts

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env tsx
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { buildPackage } from './utils/buildPackage';
+import { vanillaExtractPlugin } from '@vanilla-extract/esbuild-plugin';
+import autoprefixer from 'autoprefixer';
+import postcss from 'postcss';
+
+buildPackage({
```

Comment:
this build file is specific for `dapp-kit` because eventually we'll want to customize the build process further for things like code splitting themes, post-processing CSS, etc